include/dxc/dxcapi.h

```diff
@@ -41,12 +41,20 @@ struct IMalloc;
 
 struct IDxcIncludeHandler;
 
+/// \brief Typedef for DxcCreateInstance function pointer.
+///
+/// This can be used with GetProcAddress to get the DxcCreateInstance function.
+/// </remarks>
```

Comment:
This uses multiple inline documentation styles, and I'm not sure why this would have `</remarks>` without any matching begin tag.

Shouldn't we stick with one style?  If so, which one should we use from now on?  Doxygen style would be consistent with the llvm/clang code base.

Reply:
This is just a mistake, I'll fix it.